src/sql-editor/standardSql/standardSQLCompletionItemProvider.test.ts

```diff
@@ -0,0 +1,38 @@
+import { Monaco } from '@grafana/ui';
+import { getStandardSQLCompletionProvider } from './standardSQLCompletionItemProvider';
+import { SQLMonarchLanguage } from './types';
+
+describe('standardSQLCompletionItemProvider', () => {
+  describe('should include completion items based on the provided custom language', () => {
+    const language: SQLMonarchLanguage = {
+      tokenizer: {},
+      builtinFunctions: ['SUM', 'AVG'],
+      logicalOperators: ['AND', 'OR'],
+      comparisonOperators: ['=', '!='],
+    };
+    const completionProvider = getStandardSQLCompletionProvider({} as Monaco, language);
+    it('should use functions from language', () => {
+      expect(completionProvider.supportedFunctions().map((f) => f.name)).toEqual(language.builtinFunctions);
+    });
+    it('should combine ', () => {
+      expect(completionProvider.supportedOperators().map((o) => o.operator)).toEqual(
+        language.comparisonOperators.concat(language.logicalOperators)
+      );
+    });
+  });
+
+  describe('should include completion items based on the provided monaco registry language', () => {
```

Comment:
hey sorry how are these 2 tests different from the first? I'm sort of missing it I think, should the tests above have a language id or something?

Reply:
they are different - the first test demonstrates how completion items are picked from a custom language, and the second test demonstrates how completion items are picked from a language in the monaco registry. Language id is not important in this test, but I think you're right it makes the tests more readable so I'll add that, thanks!